data-manager/data-manager.js

```diff
@@ -1,12 +1,26 @@
-const doctors = require("../data/doctors-records");
-const patients = require("../data/patient-records");
+const doctorsRecords = require("../data/doctors-records");
+const patientsRecords = require("../data/patient-records");
 class DataManager {
+  getDateBase(string) {
```

Comment:
Nice looking methods . But "**_string_**" as a parameter denotes the data type of the argument rather than the actual purpose of the parameter